Gemfile

```diff
@@ -5,9 +5,6 @@ ruby "2.3.1"
 gem 'rails', '4.2.5'
 gem 'rails-api'
 gem 'spring', :group => :development
-gem 'omniauth-oauth2', '~> 1.3.1'
-gem 'omniauth-slack'
-gem "omniauth-google-oauth2"
```

Comment:
The idea is that we solely depend on the Andela authentication just like every other Andela services. Do you think there will be a case where this implementation will turn us down in the long run?